src/katsdpcontroller/tasks.py

```diff
@@ -102,11 +102,13 @@
 class CaptureBlockState(scheduler.OrderedEnum):
     """State of a single capture block."""
 
-    INITIALISING = 0  # Only occurs briefly on construction
-    CAPTURING = 1  # capture-init called, capture-done not yet called
-    BURNDOWN = 2  # capture-done returned, but real-time processing still happening
-    POSTPROCESSING = 3  # real-time processing complete, running batch processing
-    DEAD = 4  # fully complete
+    # fmt: off
+    INITIALISING = 0         # Only occurs briefly on construction
+    CAPTURING = 1            # capture-init called, capture-done not yet called
+    BURNDOWN = 2             # capture-done returned, but real-time processing still happening
+    POSTPROCESSING = 3       # real-time processing complete, running batch processing
+    DEAD = 4                 # fully complete
+    # fmt: on
```

Comment:
I mean it's nice to have the comments lined up like that, but I don't really mind how black does it. To me the `fmt` comments above and below are almost more unsightly.

I take it that this was for aesthetic reasons, and not functional ones.
As such, this is merely my own opinion, and has no real bearing on the merge at all.

Reply:
Yes, purely aesthetic. I'll revert it; there are other similar places where black ruined the alignment and I didn't get around to doing this, so it's currently not consistent.